app/experimenter/experiments/views.py

```diff
@@ -246,3 +246,7 @@ def view_fn(request, exception):
             return response
 
         return view_fn
+
+
+class ExperimentReportView(TemplateView):
+    template_name = "reporting.html"
```

Comment:
this is still a little wonky, i tried reporting/reporting.html and reporting-ui/reporting.html and the template paths it made to find the template didn't work out.
did I do something wrong in settings.py?